template/parser/parser.go

```diff
@@ -225,6 +225,24 @@ func (p *Parser) parseUnaryExpr() ast.Expr {
 			Arg:        p.parseExpr(),
 			Rparen:     p.expect(token.RPAREN),
 		}
+	case token.COALESCE:
+		pos := p.pos
+		p.next()
+
+		coaleacePos := pos
+		lparen := p.expect(token.LPAREN)
+		maybeUndef := p.parseExpr()
+		p.expect(token.COMMA)
+		default_ := p.parseExpr()
```

Comment:
I prefer to avoid variable names that end with an underscore.
```suggestion
		defaultVal := p.parseExpr()
```